src/OrchardCore.Build/Dependencies.props

```diff
@@ -51,7 +51,7 @@
       See https://github.com/OrchardCMS/OrchardCore/pull/16057 for more information.
```

Comment:
Let's remove the entire comment: since this package covers both the dependencies used by the OIDC handler and by OpenIddict, we shouldn't end up in a case where there's an inconsistent version in the graph.

/cc @Piedone